### ISSUE TITLE
** (ArgumentError) lib file eqc/include/eqc_mocking_api.hrl could not be found

### ISSUE THREAD

**Issue by wojtekmach** [Quviq/eqc_ex#3]
Hi, I tried using this library and got this error:

```
$ mix test

== Compilation error on file lib/eqc/mocking.ex ==
** (ArgumentError) lib file eqc/include/eqc_mocking_api.hrl could not be found
    (elixir) lib/record/extractor.ex:42: Record.Extractor.from_lib_file/1
    (elixir) lib/record/extractor.ex:13: Record.Extractor.extract/2
    lib/eqc/mocking.ex:25: (module)
    (stdlib) erl_eval.erl:657: :erl_eval.do_apply/6

==> eqc_ex
could not compile dependency eqc_ex, mix compile failed. You can recompile this dependency with `mix deps.compile eqc_ex` or update it with `mix deps.update eqc_ex`
```

Here's a repo to reproduce: https://github.com/wojtekmach/practice/tree/master/elixir/eqc_ex_example

It's probably something really simple (e.g. downloading eqcmini and setting some env variables) and it would be great if this could be documented in the README.

**Comment by ThomasArts** [comment]
Hi Wojtek

Thanks for the reported error. In order to use eqc_ex, you indeed need to have a QuickCheck version installed. There is no free version with eqc_mocking in it, thus it won't work.
After our holidays, we will document this and probably add some nice warning instead of "kaboom".

Regards
Thomas

---

Thomas Arts
CTO 
Quviq AB

thomas.arts@quviq.com
tel: +46 70 43 88567
http://www.quviq.com/

**Comment by remiq** [comment]
Hello Wojtek,

I've tried the same and I solved it by commenting eqc_mocking imports in eqc_ex source code. I've documented it here: https://github.com/remiq/elixir_quickcheck_example

Hello Tomas,

It still crashes without any helpful hint.

**Comment by ThomasArts** [comment]
Hi Wojtek

Thanks again for putting this to our attention. I solved it differently, but the next version 1.2.4 does compile with QuickCheck Mini. Preparing a hex package in a moment.

**Comment by wojtekmach** [comment]
@ThomasArts thanks, that fixes it!
